backfilling-data.md

```diff
@@ -0,0 +1,91 @@
+
+
+## Set account environment, load candidate pages and split into chunks
+
+The `get-versionista-metadata` script will gather a list of candidate pages that *may* have versions in the time period we are backfilling. Those pages are then output to disk in chunked files (currently 250 pages per chunk).
+
+Source the appropriate environment file for the Versionista account you need to backfill data from and **fill in the `--after` and `--before` options to the script below.** You can also change where the output is sent (`/data/versionista-backfill`), but you’ll need to make sure to change all the later instructions, too.
+
+```sh
+source .env.versionista1
+./bin/get-versionista-metadata --after '2017-10-09T02:00:00Z' --before '2017-10-18T00:00:00Z' --output /data/versionista-backfill/xxx --errors /data/versionista-backfill/errors-$VERSIONISTA_NAME.log --relative-paths /data/versionista-backfill/ --save-content --save-diffs --parallel 3 --pause-time 10000 --format json-stream
+```
+
+You should now see a set of files in `/data/versionista-backfill` named like `pages-versionista1-0.json` with increasing numbers on the end. Make sure there were no errors (output in files named `errors-versionista1.log`).
+
+
+## Scrape matching versions from candidate pages (repeat for each chunk)
+
+Check the list of files now in `/data/versionista-backfill` to see how many chunks you have to go through. Then repeat the following steps once for each chunk. Note they all use the `$CHUNK` variable created in the first line, so the only line you need to change each time is the first (set it to the number of the chunk you are running).
+
+This will pull down the actual raw page content, diffs, and version metadata for all the pages in a chunk.
+
+**Make sure to set the `--after` and `--before` options the same as you did for `get-versionista-metadata` above.**
+
+```sh
+# Increment this variable each time until you've done all the chunks
+CHUNK=0
+# Actually scrape version metadata, raw content, and diffs
+./bin/get-versionista-page-chunk --after '2017-10-09T00:00:00Z' --before '2017-10-18T00:00:00Z' --relative-paths /data/versionista-backfill/ --save-content --save-diffs --parallel 3 --pause-time 10000 --format json-stream --output /data/versionista-backfill/$VERSIONISTA_NAME/metadata-chunk-$CHUNK.json --errors /data/versionista-backfill/$VERSIONISTA_NAME/errors-page-chunk-$CHUNK.log --candidate-pages /data/versionista-backfill/pages-$VERSIONISTA_NAME-$CHUNK.json
+# Add a newline to the end of the file (should fix our JSON serialization to do this)
+echo '' >> /data/versionista-backfill/$VERSIONISTA_NAME/metadata-chunk-$CHUNK.json
+```
+
+You should now have a file and directory structure in `/data/versionista-backfill/versionista1` that resembles what you’d get when running the normal `scrape-versionista` script:
```

Comment:
Maybe change `/data/versionista-backfill/versionista1` to `/data/versionista-backfill/$VERSIONISTA_NAME` because the path will take on whatever the variable was